src/js/apps/patients/view/view_app.js

```diff
@@ -10,8 +13,8 @@ export default App.extend({
   },
   onBeforeStart({ viewId }) {
     if (this.isRestarting()) return;
-    const currentClinician = Radio.request('auth', 'currentUser');
-    this.groups = currentClinician.getGroups();
+    this.currentClinician = Radio.request('auth', 'currentUser');
```

Comment:
it doesn't look like you need to attach `currentClinician`

Reply:
so should I just do another Radio.request to get the `currentClinician` down in `beforeStart`?

Reply:
ah ha.  I see.  github collapsed 18-22 for me.